### ISSUE TITLE
RFE: please start making github releases

### ISSUE THREAD

**Issue by kloczek** [hreinecke/sg3_utils#97]
Is it possible next time on release new version make the github release to have entry on https://github.com/hreinecke/sg3_utils/releases? 🤔

I'm asking because only on make gh release is spread notification about new release to those who have set watch->releases.

More about gh releases is possible to find on
https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository
https://github.com/marketplace/actions/github-release

**Comment by doug-gilbert** [comment]
I did know about github releases. So I just issued one over at https://github.com/doug-gilbert/sg3_utils/releases . It is a cut and paste from Announce post that I sent to the linux-scsi list for sg3_utils 1.48 .

**Comment by kloczek** [comment]
So this rep is not `sg3_utils` primary repo? 🤔

**Comment by doug-gilbert** [comment]
No. A subversion server running on a RPi in my basement is the primary repo!
Long before I had a github account the owner of this account offered to set up a github account to mirror sg3_utils, amongst other things. That was 2007 and at the time a lot of manual work was involved. Over time "git svn ..." improved and I didn't like the attitude: "throw away svn and use git". As github became a lot more useful, almost essential for a programmer, I got my own github account.
Now, updating my sg3_utils github mirror is just 3 commands: 'git svn rebase ; git push ghub --all ; git push ghub --tags'.
Getting this mirror updated requires its owner to mirror my mirror. He just went through that exercise recently.
As I write, this repo is one commit behind my repo. That commit is only significant if code relies on up-to-date version descriptors.
Finally if you read the Issues entry you will see that I respond to most of them and if the issue has already been fixed in my repo, then I will supply a link to my github repo.
